docs/guides/specify-runtime-options.mdx

```diff
@@ -8,16 +8,12 @@ description: Specify options when building with Qiskit Runtime primitives
 
 You can use options to customize the Estimator and Sampler primitives. This section focuses on how to specify Qiskit Runtime primitive options. While the interface of the primitives' `run()`  method is common across all implementations, their options are not. Consult the corresponding API references for information about the [`qiskit.primitives`](/api/qiskit/primitives#primitives) and [`qiskit_aer.primitives`](https://qiskit.github.io/qiskit-aer/apidocs/aer_primitives.html) options.
 
-Options are specified differently depending on whether you are using the V1 or V2 interface.
-
-## V2 primitives
-
-Options are specified differently in the V2 primitives in these ways:
+Notes about specifying options in the primitives:
 
 - `SamplerV2` and `EstimatorV2` have separate options classes. You can see the available options and update option values during or after primitive initialization.
```

Comment:
Do these need to be `Sampler` and `Estimator`?

Reply:
> Do these need to be `Sampler` and `Estimator` (without `V2`)?

I thought these were the class names and needed to keep the V2.

Reply:
I think we were keeping the V2 for a bit longer to avoid confusion with people using old versions of qiskit-ibm-runtime

Reply:
I'm asking because I see both `Sampler`/`SamplerV2` and `Estimator`/`EstimatorV2` being used. For example

<img width="806" alt="image" src="https://github.com/user-attachments/assets/829ab255-417f-4903-8747-8a382c7d3cd1">

Reply:
I hopefully addressed this in my latest commit:
- Only use code when referring to actual class names
- The class names are xxxV2.
- There are still some OK uses of the old class names in the migration guides.  Hopefully I didn't get overzealous and remove any.